### DISCUSSION TITLE
Image-Focuspoint

### DISCUSSION THREAD

**Discussion by machtelik81** [directus/directus#10082] [Feature Requests]
Hi there,
are you thinking of setting up a focuspoint feature for images? A focuspoint is the focus of an image, for example a person is not in the centre of an picture but more on the side. With setting an focus point it would be possible to crop that picture on the fly, over the API, with the right dimensions needed for the needed use case, instead of uploading many different image-dimensions to directus.

The idea is that over the UI it is possible for users to set a X and Y starting position, where the focus of that certain image exists. Over the API it would be possible to ask for that set up focuspoint and crop around that focuspint. Therefore it would be sure, that the main message of an picture is safe on every dimension.

![Focuspoint-Master](https://user-images.githubusercontent.com/86186738/143574696-124d0da2-2826-49e5-8da8-18dd4ba38106.jpg)
![Focuspoint-Cropped](https://user-images.githubusercontent.com/86186738/143574702-cf85a69f-3958-4155-9d2c-50ab50eca5b9.jpg)

**Comment by benhaynes** [comment]
Definitely a good idea, and here is the same Feature Request from before: https://github.com/directus/directus/discussions/7901

**Comment by wc-matteo** [comment]
Love this. The focus point could also be initially determined by using computer vision

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
